src/server.c

```diff
@@ -5371,20 +5468,22 @@ sds genRedisInfoString(const char *section) {
     }
 
     /* Modules */
-    if (allsections || defsections || !strcasecmp(section,"modules")) {
+    if (!server.sentinel_mode && (all_sections || (dictFind(section_dict,"modules") != NULL))) {
```

Comment:
according to my proposal, this would be:
```suggestion
    if (!server.sentinel_mode && (all_sections || (dictFind(section_dict,"modules_list") != NULL))) {
```
and `modules_list` will be added to the default sections.